Fixes #1154

wojtek0806 opened the pull request:
Problem:
Policy Templates Endpoint was missing

Analysis:
Added policy template endpoint

Tests:
Flake8
Unit
Functional

caphrim007 commented:
@wojtek0806 here is the patch that fixes the errors i saw for 11.6.0

https://gist.github.com/caphrim007/145826c42595fa0e351c21fb21308f16

wojtek0806 commented:
I will clean up any travis problems and will disable those failing tests and open up an issue to rework the ASM policies to split it into smaller bits. So we can merge this. @caphrim007

wojtek0806 committed:
Fixes #1154

Problem:
Policy Templates Endpoint was missing

Analysis:
Added policy template endpoint

Tests:
Flake8
Unit
Functional

caphrim007 commented:
lgtm
